subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -4090,7 +4090,7 @@ SHELL_SUBCMD_ADD((wifi), connect, NULL,
 		 "[-c --channel]: Channel that needs to be scanned for connection. "
 		 "0:any channel.\n"
 		 "[-b, --band] 0: any band (2:2.4GHz, 5:5GHz, 6:6GHz]\n"
-		 "[-p, --psk]: Passphrase (valid only for secure SSIDs)\n"
+		 "[-p, --passphrase]: Passphrase (valid only for secure SSIDs)\n"
```

Comment:
a bit tricky, WPA2 calls is psk, WPA3 calls it passphrase, maybe we should use password as a generic term?

Reply:
Yes, though most usecases are with short options, but yeah long options will break, and either way it's not completely correct term, so, may be leave it as is? or leave the option as `--psk` and elaborate help text?